public/js/framework.js

```diff
@@ -1,17 +1,15 @@
 /**
```

Comment:
This file should go to public/js/**plb**

We want to separate our JS files from vendor files (that will probably go to public/js/vendor)